src/commands/properties/set.ts

```diff
@@ -42,9 +42,9 @@ export class PropertiesSetCommand extends UpdatePropertiesCommand {
                 throw new Error(`Property parsing error. Should match key:value ${prop}`);
             }
             const [key, value] = keyValue;
-            const keyMatcher = key.match(/[a-zA-Z_0-9]+/);
+            const keyMatcher = key.match(UNS_NFT_PROPERTY_KEY_REGEX);
             if (!keyMatcher || keyMatcher[0] !== key) {
-                throw new Error(`Property key ${key} should match [a-zA-Z_0-9]+ pattern`);
+                throw new Error("property key does not match the expected format");
```

Comment:
Previous error message was much more interesting: it displayed the expected format to the user (great product!).
If possible, restore the previous error message with the property name and the new pattern :muscle: